README.md

```diff
@@ -24,6 +23,14 @@ For a development install:
     jupyter labextension install @jupyter-widgets/jupyterlab-manager
     jupyter labextension install @jupyter-widgets/jupyterlab-sidecar
 
+
```

Comment:
I'm wondering if for the installation instructions, we should separate into a User/Dev install? I don't think users will need to install the linting features for VSCode or Atom. Maybe we can do a separate section for '#Development Installation' inside the README?

Reply:
I guess the README was updated again in the other PR, so you can ignore this comment.